tutorial/10_key_features/004_distributed.py

```diff
@@ -4,7 +4,7 @@
 Easy Parallelization
 ====================
 
-It's straightforward to parallelize ``optuna.optimize()``.
+It's straightforward to parallelize :func:`optuna.study.Study.optimize`.
```

Comment:
[question] Is a word such as `using` or `by` needed between `parallelize` and `:func:...`?